uploaders/GitHubUploader.js

```diff
@@ -42,41 +43,52 @@ class GitHubUploader {
     const { createReadStream, filename, mimetype, encoding } = await file;
     const stream = createReadStream();
 
-    return new Promise(function (resolve, reject) {
-      gm(stream, filename)
-        .resize(2000, null, '>')
-        .quality(60)
-        .stream(function (err, stdout) {
-          if (err) {
-            reject(new Error(`${filename} errored with ${err}!`));
-          }
-
-          const bufs = [];
-          stdout.on('data', function (d) {
-            bufs.push(d);
+    const streamArray = ['outputStream', 'outputThumbStream'].map(function (i) {
+      return new Promise(function (resolve, reject) {
+        let w = 2000;
+        let q = 60;
+        if (i === 'outputStream') {
+          w = 2000;
+          q = 60;
+        } else {
+          w = 500;
+          q = 50;
+        }
+        gm(stream, filename)
+          .resize(w, null, '>')
+          .quality(q)
+          .stream(function (err, stdout) {
+            if (err) {
+              reject(new Error(`${filename} errored with ${err}!`));
+            }
+
+            const bufs = [];
+            stdout.on('data', function (d) {
+              bufs.push(d);
+            });
+
+            stdout.on('end', function () {
+              this[i] = Buffer.concat(bufs);
+              resolve({ filename, mimetype, encoding, stream: this[i] });
+            });
           });
-
-          stdout.on('end', function () {
-            this.outputStream = Buffer.concat(bufs);
-            resolve({ filename, mimetype, encoding, stream: this.outputStream });
-          });
-        });
+      });
     });
-  }
 
-  async deleteFileAfterCommit(file) {
-    return new Promise(function (resolve) {
-      console.info(`${file} committed and removed successfully`);
-      resolve(true);
-    });
+    return streamArray;
   }
 
-  async uploadToRepo(parent, { file }) {
+  async uploadToRepo(parent, { file, type = 'library' }) {
```

Comment:
You could console.log it or something if you want to keep it around in the code,  I guess.

Reply:
removed